src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -189,12 +189,26 @@ def test_method_with_enum_output_type_method(device_info):
         #will have to update after https://github.com/ni/nimi-python/issues/128 fixed
         assert session.read_status()[1] == 4        
         
-		
+
 def test_writeonly_attribute(device_info):
     with nidmm.Session(device_info['name']) as session:
         try:
             session.channel_count = 5
+            assert False
         except nidmm.Error as e:
             assert e.code == -1074135027 #Error : Attribute is read-only.
 
 
+def test_invalid_value_attribute(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        try:
+            session.settle_time = -5
+            assert False
+        except nidmm.Error as e:
+            assert e.code == -1074135024 #Error : Invalid value for parameter or property.
+
+
+def test_ViInt32_output_function(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        assert isinstance(session.get_cal_count(0), int) #TODO(bhaswath): when enum for cal_type is added, replace 0 with Internal or External
```

Comment:
This is a more general question. Do we want to include system tests in the flake8 testing? I.e. there are comments in this file (new & existing) that wouldn't pass.